modules/api/src/main/java/org/eclipse/xpanse/api/controllers/IsvServiceDeployApi.java

```diff
@@ -0,0 +1,100 @@
+import java.util.Objects;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.xpanse.modules.deployment.DeployService;
+import org.eclipse.xpanse.modules.models.service.common.enums.Category;
+import org.eclipse.xpanse.modules.models.service.common.enums.Csp;
+import org.eclipse.xpanse.modules.models.service.deploy.enums.ServiceDeploymentState;
+import org.eclipse.xpanse.modules.models.service.query.ServiceQueryModel;
+import org.eclipse.xpanse.modules.models.service.view.ServiceVo;
+import org.eclipse.xpanse.modules.models.servicetemplate.enums.ServiceHostingType;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * REST interface methods for processing OCL.
+ */
+@Slf4j
+@RestController
+@RequestMapping("/xpanse")
+@CrossOrigin
+@Secured({ROLE_ISV})
+public class IsvServiceDeployApi {
+
+    @Resource
+    private DeployService deployService;
+
+    /**
+     * List all deployed services by a user of Isv.
+     *
+     * @return list of all services deployed by a user.
+     */
+    @Tag(name = "Service", description = "APIs to manage the service instances")
+    @Operation(description = "List all deployed services by a user.")
+    @GetMapping(value = "/isv/services",
+            produces = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.OK)
+    public List<ServiceVo> listDeployedServicesOfIsv(
+            @Parameter(name = "categoryName", description = "category of the service")
+            @RequestParam(name = "categoryName", required = false) Category category,
+            @Parameter(name = "cspName", description = "name of the cloud service provider")
+            @RequestParam(name = "cspName", required = false) Csp csp,
+            @Parameter(name = "serviceName", description = "name of the service")
+            @RequestParam(name = "serviceName", required = false) String serviceName,
+            @Parameter(name = "serviceVersion", description = "version of the service")
+            @RequestParam(name = "serviceVersion", required = false) String serviceVersion,
+            @Parameter(name = "serviceState", description = "deployment state of the service")
+            @RequestParam(name = "serviceState", required = false)
+                    ServiceDeploymentState serviceState) {
+        ServiceQueryModel query =
+                getServiceQueryModel(category, csp, serviceName, serviceVersion, serviceState);
+        return this.deployService.listDeployedServices(query);
```

Comment:
here we must return only services belonging to the ISV to which the logged in user is belonging to.